src/main/java/com/bootcamp/demo/service/ILoginService.java

```diff
@@ -0,0 +1,15 @@
+package com.bootcamp.demo.service;
+
+import com.bootcamp.demo.model.User;
+
+/**
+ * LoginService interface used for registration, login and logout
+ */
+public interface ILoginService {
+
+    boolean registerUser(User new_user);
```

Comment:
Please use standard Java naming conventions: "newUser" instead of "new_user".
Some javadoc documentation for the API operations would be useful.